runtime/westend/src/lib.rs

```diff
@@ -1835,3 +1859,42 @@ mod remote_tests {
 		ext.execute_with(|| Runtime::on_runtime_upgrade());
 	}
 }
+
+mod init_state_migration {
```

Comment:
We have sudo on westend. We can set any of these manually as well FYI.

Reply:
In fact, I think it is better to start it manually via sudo than bundling it with the migration.

Reply:
I would like it to run the exact same way as it will be on kusama and polkadot, so without sudo